.vscode/extensions.json

```diff
@@ -8,7 +8,8 @@
     "pflannery.vscode-versionlens",
     "editorconfig.editorconfig",
     "prisma.prisma",
-    "graphql.vscode-graphql"
+    "graphql.vscode-graphql",
+    "styled-components.vscode-styled-components"
```

Comment:
If you're working with styled components, this extension is a must!

Reply:
+1 to that!